sos/report/plugins/foreman_installer.py

```diff
@@ -89,12 +89,12 @@ def setup(self):
 
         self.add_file_tags({
             '/var/log/foreman-installer/satellite.log':
-                ['foreman_satellite_log' 'satellite_installer_log'],
+                ['foreman_satellite_log+satellite_installer_log'],
```

Comment:
Addresses "W1404: Implicit string concatenation found in list (implicit-str-concat)"

Reply:
There is missing coma :)

Currently - and your PR keeps that behaviour - we get `manifest.json`:

                        {
                            "specification": "/var/log/foreman-installer/satellite.log",
                            "files_copied": [
                                "var/log/foreman-installer/satellite.log"
                            ],
                            "tags": [
                                "foreman_satellite_logsatellite_installer_log"
                            ]
                        },
                        {
                            "specification": "/var/log/foreman-installer/capsule.log",
                            "files_copied": [
                                "var/log/foreman-installer/capsule.log"
                            ],
                            "tags": [
                                "capsule_logcapsule_installer_log"
                            ]
                        }
    ..
            "tag_summary": {
                "capsule_logcapsule_installer_log": [
                    "var/log/foreman-installer/capsule.log"
                ],
                "foreman_satellite_logsatellite_installer_log": [
                    "var/log/foreman-installer/satellite.log"
                ]
            }

while we should get rather:

                        {
                            "specification": "/var/log/foreman-installer/satellite.log",
                            "files_copied": [
                                "var/log/foreman-installer/satellite.log"
                            ],
                            "tags": [
                                "foreman_satellite_log",
                                "satellite_installer_log"
                            ]
                        },
                        {
                            "specification": "/var/log/foreman-installer/capsule.log",
                            "files_copied": [
                                "var/log/foreman-installer/capsule.log"
                            ],
                            "tags": [
                                "capsule_log",
                                "capsule_installer_log"
                            ]
                        }
    ..
            "tag_summary": {
                "capsule_installer_log": [
                    "var/log/foreman-installer/capsule.log"
                ],
                "capsule_log": [
                    "var/log/foreman-installer/capsule.log"
                ],
                "foreman_satellite_log": [
                    "var/log/foreman-installer/satellite.log"
                ],
                "satellite_installer_log": [
                    "var/log/foreman-installer/satellite.log"
                ]
            }

Git blame shows some change due to https://github.com/sosreport/sos/commit/77614068aef89ad13492f819beabbad07a53dd47 and https://github.com/sosreport/sos/commit/962960c5e15e7349067a920aca7c8eae157e67d7 prior that. I doubt the current tags are really used (or @ryan-blakley or @TurboTurtle  is aware of their usage?). So I would not hesitate to change them to something more meaningful.

I would suggest:

    self.add_file_tags({
            '/var/log/foreman-installer/satellite.log':
                ['foreman_installer_log', 'satellite_installer_log'],
            '/var/log/foreman-installer/capsule.log':
                ['foreman_installer_log', 'capsule_installer_log'],
    })

as both logs are foreman_installer ones, one for installing Satellite, one for installing Capsule.

Reply:
Thanks :) I'll modify as you suggested.